the-blue-alliance-ios/View Elements/Match/MatchSummaryView.swift

```diff
@@ -144,30 +156,67 @@ class MatchSummaryView: UIView {
     private func teamLabel(for teamKey: String, baseTeamKey: String?, dq: Bool) -> UILabel {
         let text: String = "\(Team.trimFRCPrefix(teamKey))"
         let isBold: Bool = (teamKey == baseTeamKey)
-
+        
         return label(text: text, isBold: isBold, isStrikethrough: dq)
     }
+    
+    private func teamButton(for teamKey: String, baseTeamKey: String?, dq: Bool) -> UIButton {
+        let text: String = "\(Team.trimFRCPrefix(teamKey))"
+        let isBold: Bool = (teamKey == baseTeamKey)
+
+        return button(text: text, isBold: isBold, isStrikethrough: dq)
+    }
+    
+    private func button(text: String, isBold: Bool, isStrikethrough: Bool = false, isTeam: Bool = true) -> UIButton {
+        let newButton = UIButton(type: .system)
+        newButton.setTitle(text, for: [])
+        
+        // set color of the label to black like default
+        let buttonColor = UIColor.primaryBlue
+        newButton.setTitleColor(buttonColor, for: .normal)
+        // change color to gray when user hits the button
+        newButton.setTitleColor(buttonColor.withAlphaComponent(0.8), for: .highlighted)
```

Comment:
The default color is a light blue, which didn't fit any color that's currently used in the app. It shouldn't be black since there needs to be an indicator that it is tappable. I chose the primary blue color that's used throughout the app to avoid adding in a completely new color but we can definitely change that accordingly.

Reply:
The primary blue for the base color seems fine - but is there a reason we need to override the tint color stuff? Why not let the system handle that by default?